src/index.ts

```diff
@@ -0,0 +1,3 @@
+import { RedisPubSub as redis } from "./redis-pub-sub";
+
+export const RedisPubSub = redis;
```

Comment:
I think we can just do this:

```suggestion
export { RedisPubSub } from "./redis-pub-sub";
```

Reply:
it worked!